packages/workflow-app-textedit/Readme.md

```diff
@@ -0,0 +1,33 @@
+# TextEdit workflow app
+
+## Installation
+
+```
+npm i workflow-app-textedit
+```
+
+## Usage
+
+```
+import TextEdit from 'workflow-app-textedit';
+
+export default {
+  name: 'workflow-example',
+  type: 'workspace',
+  children: [
+    {
+      type: 'layout',
+      layout: 'splith',
+      percent: 1.0,
+      children: [
+        {
+          ...TextEdit,
+          cwd: __dirname,
```

Comment:
We need to standardize these props. They are valid for a Texted or, but I cannot see that they are supported by your implementation of of TextEdit. 

Today we have `file`, and I have previously implemented `folder` as opposed to `cwd` for editors. I think that `filename` has a too narrow use case. What about ` file` and`foldrr` where the `file`is either absolute or relative to the `folder`? `cwd` just doesn't fell right for an editor to me, but it might give us some consistency with `Terminals`

Reply:
This is me just copy pasting stuff, will change it 😅